src/backend/distributed/sql/udfs/get_nodeid_for_gpid/latest.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE FUNCTION pg_catalog.get_nodeid_for_gpid(global_pid bigint)
+    RETURNS integer
+    LANGUAGE C STABLE
```

Comment:
These all were `strict` functions, why did we remove that? I think they are useful anyway.

Reply:
STABLE is about volatility https://www.postgresql.org/docs/14/xfunc-volatility.html
STRICT is about NULL inputs: `STRICT indicates that the function always returns null whenever any of its arguments are null. If this parameter is specified, the function is not executed when there are null arguments; instead a null result is assumed automatically.`